src/pages/api/auth/signup.tsx

```diff
@@ -0,0 +1,53 @@
+//Server-side code
+import express from 'express'
+import bodyParser from 'body-parser'
+
+const app = express()
```

Comment:
express config should never be used for NextJS - It has far more efficient ways of doing things - especially with TRPC type safety and cached requests